tasks/copy_to_glacier/README.md

```diff
@@ -28,42 +28,91 @@ Note that this must be done for _each_ collection configured. If this list is em
 The following steps assume you are using a version of Python compliant with 3.7 (`pip` comes with current versions).
 
 ```
-python -m venv venv
+cd tasks\copy_to_glacier
+rm -rf build
+rm -rf venv
+mkdir build
+python3 -m venv venv
 source venv/bin/activate
-pip install --upgrade pip       # Upgrade pip
-pip install -r requirements.txt # requirements-dev.txt if you're testing/developing
+pip install --upgrade pip
+pip install -q -t build -r requirements.txt --trusted-host pypi.org --trusted-host pypi.org --trusted-host files.pythonhosted.org
+deactivate
+cp *.py build/
+cd build
+zip -r ../copy_to_glacier.zip .
+cd ..
+
 ```
+An example of building the `copy_to_glacier` lambda package can be found in `tasks/copy_to_glacier/bin/build.sh`.
+
+### Testing copy_to_glacier
+
+To run unit tests for **copy_to_glacier**, run the `bin/run_tests.sh` script from the
+`tasks/copy_to_glacier` directory. Ideally, the tests should be run in a docker
+container. The following shows setting up a container to run the tests.
+
+```bash
+# Invoke a docker container in interactive mode.
+user$ docker run \
+      -it \
+      --rm \
+      -v /path/to/cumulus-orca/repo:/data \
+      amazonlinux:2 \
+      /bin/bash
 
-An explicit example of building the lambda package can be found in `/bin/build_tasks.sh`.
+# Install the python development binaries
+bash-4.2# yum install python3-devel
+
+# In the container cd to /data
+bash-4.2# cd /data
+
+# Go to the task
+bash-4.2# cd tasks/copy_to_glacier/
+
+# Run the tests
+bash-4.2# bin/run_tests.sh
+```
+
+Note that Bamboo will run this same script via the `bin/run_tests.sh` script found
+in the cumulus-orca base of the repo.
 
 ## Deployment
 
-Upload the zip file to AWS (either through the cli or console). Alternatively, `modules/lambdas/main.tf` shows an example of deploying this lambda through Terraform.
+The `copy_to_glacier` lambda function can be deployed using terraform using the example shown in  `modules/lambdas/main.tf` file.
 
 ```
 resource "aws_lambda_function" "copy_to_glacier" {
-  function_name    = "${var.prefix}_copy_to_glacier"
+  ## REQUIRED
```

Comment:
I would probably remove the code here and reference the resource to look at in the terraform lamdas/main.tf file.

Reply:
removed